Gemfile.lock

```diff
@@ -1,47 +1,36 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (5.2.0)
-      concurrent-ruby (~> 1.0, >= 1.0.2)
-      i18n (>= 0.7, < 2)
-      minitest (~> 5.1)
-      tzinfo (~> 1.1)
-    backports (3.11.3)
-    concurrent-ruby (1.0.5)
-    i18n (1.0.1)
-      concurrent-ruby (~> 1.0)
-    minitest (5.11.3)
-    multi_json (1.13.1)
-    mustermann (1.0.2)
-    puma (3.11.4)
-    rack (2.0.5)
-    rack-protection (2.0.3)
+    multi_json (1.15.0)
+    mustermann (1.1.1)
+      ruby2_keywords (~> 0.0.1)
+    nio4r (2.5.4)
+    puma (5.1.1)
+      nio4r (~> 2.0)
+    rack (2.2.3)
+    rack-protection (2.1.0)
       rack
-    sinatra (2.0.3)
+    ruby2_keywords (0.0.2)
+    sinatra (2.1.0)
       mustermann (~> 1.0)
-      rack (~> 2.0)
-      rack-protection (= 2.0.3)
+      rack (~> 2.2)
+      rack-protection (= 2.1.0)
       tilt (~> 2.0)
-    sinatra-contrib (2.0.3)
-      activesupport (>= 4.0.0)
-      backports (>= 2.8.2)
+    sinatra-contrib (2.1.0)
       multi_json
       mustermann (~> 1.0)
-      rack-protection (= 2.0.3)
-      sinatra (= 2.0.3)
-      tilt (>= 1.3, < 3)
-    thread_safe (0.3.6)
-    tilt (2.0.8)
-    tzinfo (1.2.5)
-      thread_safe (~> 0.1)
+      rack-protection (= 2.1.0)
+      sinatra (= 2.1.0)
+      tilt (~> 2.0)
+    tilt (2.0.10)
 
 PLATFORMS
-  ruby
+  x86_64-linux
```

Comment:
I ran `bundle install` locally and it changed to `ruby` again. I'm not sure why this is happening, I'm assuming your ruby installation is causing this to change.

This doesn't seem to be a problem, I ran `bundle platform` and it seems both are compatible, but worths mentioning

```
Your platform is: x86_64-darwin19

Your app has gems that work on these platforms:
* ruby
* x86_64-linux
```